src/chapters/git/studio_rev_1.rst

```diff
@@ -64,14 +69,25 @@ Step 1: Create a New Local Repository
    part of the commands.
 
```

Comment:
I think a role tag would be good here:  **Control and Pilot** and for step 2 (line 173)

Reply:
Good idea.  I added global role tags at the start of steps 1 and 2.